src/Common/SettingsChanges.cpp

```diff
@@ -46,4 +46,29 @@ Field * SettingsChanges::tryGet(std::string_view name)
     return &change->value;
 }
 
+bool SettingsChanges::insertSetting(std::string_view name, const Field & value)
+{
+    if (std::find_if(begin(), end(), [&name](const SettingChange & change) { return change.name == name; }) != end())
```

Comment:
We should just copy string view and do not take reference in lambda `[&name]`.

Reply:
it's how it used in existing code above and I don't see it's anyhow important here